packages/salesforcedx-vscode-lwc/src/commands/commandConstants.ts

```diff
@@ -6,4 +6,5 @@
  */
 
 export const DEV_SERVER_BASE_URL = 'http://localhost:3333';
-export const DEV_SERVER_PREVIEW_ROUTE = `${DEV_SERVER_BASE_URL}/lwc/preview`;
+export const DEV_SERVER_PREVIEW_ROUTE = 'preview';
```

Comment:
Dropping the 'lwc' part. Right now devserver supports both. Once majority customers migrate to this version of VSCode, we can drop the support for the old path.